org.lflang.tests/src/org/lflang/tests/Configurators.java

```diff
@@ -46,24 +46,35 @@
     }
 
     /**
-     * Configure the given test by setting its `threads` target property to 0.
+     * Configure the given test to use single-threaded execution.
+     *
+     * For targets that provide a threaded and an unthreaded runtime,
+     * this configures using the unthreaded runtime. For targets that
+     * do not distinguish threaded and unthreaded runtime, the number
+     * of workers is set to 1.
      *
      * @param test The test to configure.
      * @return True if successful, false otherwise.
      */
     public static boolean useSingleThread(LFTest test) {
-        test.context.getArgs().setProperty("threads", "0");
+        test.context.getArgs().setProperty("threading", "false");
+        test.context.getArgs().setProperty("workers", "1");
```

Comment:
Isn't the workers property unnecessary/meaningless when `threading` is set to false?